docs/the-new-architecture/use-app-template.md

```diff
@@ -3,5 +3,61 @@ id: use-app-template
 title: Using the New App Template
 ---
 
-This section describes the new app template generated by the React Native CLI and how to use it properly.
-It should explain how to create a new app, which command should be issued to install the dependencies and to run it on the different platforms.
+This page will help you create a new React Native app that uses the new architecture.
+
+## Installing dependencies
+
+Start by setting up your development environment as described in [Getting Started](getting-started.md), under "React Native CLI Quickstart". Stop when you reach "React Native Command Line Interface", and resume here.
+
+<h3>React Native Command Line Interface</h3>
+
+React Native has a built-in command line interface. Rather than install and manage a specific version of the CLI globally, we recommend you access the current version at runtime using `npx`, which ships with Node.js. With `npx react-native <command>`, the current stable version of the CLI will be downloaded and executed at the time the command is run.
+
+<h2>Creating a new application</h2>
+
+> If you previously installed a global `react-native-cli` package, please remove it as it may cause unexpected issues.
+
+React Native has a built-in command line interface, which you can use to generate a new project. You can access it without installing anything globally using `npx`, which ships with Node.js. Let's create a new React Native project called "AwesomeProject":
```

Comment:
These lines are pulled directly from the "Getting started" section because I noticed there's a lot of overlap in the steps, still, even beyond dependency installation. Comparing this documentation with "Getting started" leads me to a few questions:

* Is it appropriate to repeat all of this information? "Getting started" is very thorough, since it's aimed at beginners, but I would suspect many beginners to React Native will be interested in starting with the new architecture enabled.
* If we're going to be repeating information, should we make reusable markdown fragments to use in both locations? There are a few items that would need to be different (e.g., "if integrating react-native into an existing app" link), but that seems manageable.
* "Getting Started" also includes instructions with command line examples for Windows and Linux. Do we need that support here too? (e.g., some troubleshooting examples might need to be different per-platform)
* Considering all of the above -- should this documentation perhaps also live within "Getting Started"?

Reply:
> For the time being, we can definitely add a link in the Getting Started guide to this section saying that, if you are interested in creating a new app with the New Architecture, this is the article for you.

Sounds good!

> I don't think we should cover the case of how to integrate RN and the New Architecture in an existing app in this page. 

To make sure it's clear, I just quoted that example because it was part of the page I copied from originally, so yeah, it's an example of something that would have to change if we used snippets.

> If they are related to how to run the app on those platforms, I think yes. Especially if there are difference on how the cli runs on those platforms.

Okay, that may come up for Android, depending on the troubleshooting section. If it does I'll plan to move to using Mac/Windows/Linux platform tabs like "Getting Started".

> But I'm starting thinking that those section should be splitted

To make sure I understand, are you taking about the structure of "Getting Started" here? Or this page?

> Also, the first two sentences are a repetition of the <h3>React Native Command Line Interface</h3> section and we could remove them.

My hope with creating reusable fragments was that we could actually avoid linking to "Getting Started" altogether, and instead copy over the contents of the page that are relevant. I think that might be a better experience, because:
* [there's no way to link to the specific relevant tab in "Getting Started"](https://docusaurus.canny.io/feature-requests/p/link-to-tab), so as-is, the reader is shown the tab for the default Expo CLI Quickstart.
* The directions to "stop reading at when you reach this section and come back" adds a little more friction. The reader will likely forget where to stop reading and may make a mistake and go too far, and get confused about what to do next. Not a huge issue, but perhaps less than ideal.

Reply:
> To make sure I understand, are you taking about the structure of "Getting Started" here? Or this page?

I'm talking in the `Getting Started` section. It could be split into two:
1. setup your environment
1. create your first app

For the New Architecture, we only have to do the second point, because the first one will be the same, independently from the architecture. This will also relieve you from the burden of ask the user to setup their environment.

> My hope with creating reusable fragments was that we could actually avoid linking to "Getting Started" altogether, and instead copy over the contents of the page that are relevant.

This is a good point. Another [user](https://github.com/reactwg/react-native-new-architecture/discussions/9#discussioncomment-2465185) pointed out a similar issue, so it can be a particularly good solution. But I think it works well for small paragraph: reintroducing the whole setup page could create a big article in the guide that can be intimidating.

For bigger articles, like `Setup your environment` could make sense to add it twice in the sidebar: we are still reusing the material completely and we create a logical flow to follow. What do you think?

Reply:
> For bigger articles, like Setup your environment could make sense to add it twice in the sidebar: we are still reusing the material completely and we create a logical flow to follow. 

If I'm understanding this correctly, this would mean a left sidebar that looks like:

```
Basics
Environment Setup
  Setting up the development environment
  Creating a new app
  Integration with Existing Apps
  ...
The New Architecture
  Introduction
  Why a New Architecture
  Setting up the development environment
  Creating a new app
```

One obstacle for that: I think the reason "setting up the development environment" and "creating a new app" concepts are combined in the existing "Getting Started" guide is probably because of the Expo guide — it does both at once. I'm not sure how to represent that without Expo being its own page as well, which feels less polished.

Reply:
> For the time being, we can definitely add a link in the _Getting Started_ guide to this section saying that, if you are interested in creating a new app with the New Architecture, this is the article for you.

I don't have a strong opinion here, but I believe we can avoid re-explaining how the whole CLI works and just mention the crucial steps (i.e. the `npx react-native init` and so on).

I think it's safe to assume the user has an environment setup & mention that page as a prerequisite.

Reply:
Opted for a simpler breakdown here, just linking to the existing guide and mentioning that it's a prerequisite.

If it still looks awkward, I do think the best bet is copying relevant sections of the quickstart guide to here.